conda-forge.yml

```diff
@@ -3,3 +3,5 @@ provider: {linux_ppc64le: azure, linux_aarch64: azure}
 github:
   branch_name: main
   tooling_branch_name: main
+azure:
+  store_build_artifacts: True
```

Comment:
I think we can remove this now that we have manually verified everything that we need, right?

Reply:
Rolled back this change.